src/components/molecules/About/AboutPage.tsx

```diff
@@ -1,19 +1,25 @@
-import { Box, Button, Typography } from "@mui/material";
-import { Link } from "react-router-dom";
-import { Kurt } from "../../../assets/personalInfo/personalInfo";
+import { Box, Typography } from '@mui/material'
+import { Kurt } from '../../../assets/personalInfo/personalInfo'
+import { Contact } from '../../templates/Contact'
 
 export function AboutPage() {
   const title = Kurt.title
   const description = Kurt.description
-  
+
   return (
     <>
       <Typography variant="h3">{title}</Typography>
       <Typography>{description}</Typography>
       <Box display="flex" alignContent="center" justifyContent="center">
-        {Kurt.photos.map((photo) => <img src={photo.path} alt={photo.alt} style={{height: "100%", width: "100%"}}/>)}
+        {Kurt.photos.map(photo => (
+          <img
+            src={photo.path}
+            alt={photo.alt}
+            style={{ height: '100%', width: '100%' }}
+          />
+        ))}
       </Box>
-      <Button component={Link} to="/#contact">Like my work?</Button> 
+      <Contact />
```

Comment:
👍 Super easy, ya?

Reply:
So dang easy!
Atomic structure has definitely grown on me, and is awesome